taskfile.yml

```diff
@@ -36,17 +37,27 @@ tasks:
         vars:
           SERVICE_NAME: orchestrator
 
+  proto:gen:svr:
+    desc: "Generate Go code from protobuf files for svr"
+    cmds:
+      - find ./svr/ -name '*.pb.go' -delete
+      - task: proto:wsrpc:gen
```

Comment:
Should this be `proto:gen` instead? Unless youe app is running `wsRPC` you don't need `proto:wsrpc:gen`